models/thena/bnb/thena_v1_bnb_trades.sql

```diff
@@ -20,22 +20,21 @@ WITH dexs AS
     SELECT
         t.evt_block_time                                                                AS block_time,
         t.to                                                                            AS taker,
-        ''                                                                              AS maker,
-        CASE WHEN amount0Out = '0' THEN amount1Out ELSE amount0Out END                  AS token_bought_amount_raw,
-        CASE WHEN amount0In = '0' OR amount1Out = '0' THEN amount1In ELSE amount0In END AS token_sold_amount_raw,
-        cast(NULL as double)                                                            AS amount_usd,
-        CASE WHEN amount0Out = '0' THEN f.token1 ELSE f.token0 END                      AS token_bought_address,
-        CASE WHEN amount0In = '0' OR amount1Out = '0' THEN f.token1 ELSE f.token0 END   AS token_sold_address,
+        0x                                                                              AS maker,
+        CASE WHEN amount0Out = UINT256 '0' THEN amount1Out ELSE amount0Out END                  AS token_bought_amount_raw,
+        CASE WHEN amount0In = UINT256 '0' OR amount1Out = UINT256 '0' THEN amount1In ELSE amount0In END AS token_sold_amount_raw,
+        NULL                                                           AS amount_usd,
+        CASE WHEN amount0Out = UINT256 '0' THEN f.token1 ELSE f.token0 END                      AS token_bought_address,
+        CASE WHEN amount0In = UINT256 '0' OR amount1Out = UINT256 '0' THEN f.token1 ELSE f.token0 END   AS token_sold_address,
         t.contract_address                                                              AS project_contract_address,
         t.evt_tx_hash                                                                   AS tx_hash,
-        ''                                                                              AS trace_address,
         t.evt_index
     FROM
         {{ source('thena_fi_bnb', 'pair_evt_swap') }} t
-        INNER JOIN {{ source('thena_fi_bnb', 'PairFactoryUpgradeable_evt_PairCreated') }} f
+        INNER JOIN {{ source('thena_bnb', 'PairFactoryUpgradeable_evt_PairCreated') }} f
```

Comment:
Currently there is only a decoded source `thena_fi_bnb.PairFactoryUpgradeable_evt_PairCreated` available.